File: sem2/list3/c++/kolo.cpp
#include "inc/kolo.hpp"

Kolo::Kolo(double promien) : _promien(promien) {}

double Kolo::obwod() {
    return M_PI * 2.0 * _promien;
}

double Kolo::pole() {
    return M_PI * _promien * _promien;
}

File: sem2/list3/java/zad1/Kolo.java
package sem2.list3.java.zad1;
import static java.lang.Math.*;

public class Kolo implements Figura {
    protected double _promien;

    public Kolo(double promien) {
        _promien = promien;
    }

    public double obwod() {
        return PI * 2 * _promien;
    }
    
    public double pole() {
        return PI * _promien * _promien;
    }
}